dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -94,6 +94,22 @@ When a set of routes is created in various projects, the overall set of routes
 is available to the set of routers. Each router admits (or selects) routes from
 the set of routes. By default, all routers admit all routes.
 
+As of {product-title} 3.6, routers can accept IPV6 traffic from outside the
+cluster for the following routes:
+
+* Route type 1
```

Comment:
since the change allows haproxy to conect to ipv6 interfaces it should work for all the route types